tests/fv3_conf/fv3_slurm.IN

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
This is a temporary file created from fv3_slurm.IN_platform, I suggest not to commit it.

Reply:
I was wondering why it was there. I will remove.